backend/tests/unit/services/test_abort_service.py

```
@@ -0,0 +1,310 @@
+        store = InMemoryStore()
+        user_id = str(uuid4())
+        other_user_id = str(uuid4())
+        thread_id = str(uuid4())
+        service = AbortService(user_id=user_id, store=store)
+
+        # Mock thread_service.get to return thread with different user_id in metadata
+        mock_thread = MagicMock()
+        mock_thread.metadata = {"user_id": other_user_id}
+        service.thread_service.get = AsyncMock(return_value=mock_thread)
+
+        with pytest.raises(PermissionError, match=f"Not authorized to abort thread"):
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)